yevhenii.dolmatov/task_06/nokogiri.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require 'nokogiri'
+
+# implementation of Document class
+class Document
```

Comment:
Please create separate file for each class

Reply:
Ok, no problem.